components/Stands/CompanyCardInfo.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from 'styled-components';
+import { eventTime, toDayjs } from '../../utils/time';
+import { StandCard_stand } from '../../__generated__/StandCard_stand.graphql';
+import LiveIndicator from './LiveIndicator';
+import { CompanyImg, EventTitle, standEvent, TimeSlot } from './StandCard';
+
+interface CompanyCardContentProps {
+  stand: StandCard_stand;
+  currentEvent: standEvent | null;
+}
+
+const CompanyCardContent = ({
+  stand,
+  currentEvent,
+}: CompanyCardContentProps): JSX.Element => (
+  <>
+    <FirstRow>
+      <CompanyImgContainer>
+        <CompanyImg src={stand?.company.logo ?? ''} />
+      </CompanyImgContainer>
+      <LiveIndicator active={stand?.active ?? false} />
```

Comment:
can stand be null here? and stand.active is a boolean so why you shouldn't need to explicitly pass false

Reply:
It actually can't at this point. I'll change it 👍